layout/theme.liquid

```diff
@@ -128,61 +130,96 @@
 
   {{ content_for_layout }}
 
-  <footer id="footer">
+  <footer id="footer" class="padding-top-small padding-bottom-small">
     <div class="container">
-      <div class="grid-row justify-between">
-        <div class="grid-item item-s-12 item-m-4">
-        {% if settings.contact_address != blank %}
-          <div class="font-bold">
-            {{ settings.contact_address }}
-          </div>
-        {% endif %}
-        {% if settings.hours_week != blank %}
-          <div>
-            {{ settings.hours_week }}{% if settings.hours_weekend != blank %}, {% endif %}
-          </div>
-        {% endif %}
-        {% if settings.hours_weekend != blank %}
-          <div>
-            {{ settings.hours_weekend }}
+      <div id="footer-row" class="grid-row justify-between align-items-end">
+        <div class="grid-item no-gutter item-s-12 grid-row justify-between">
+          <div class="grid-item item-s-12 item-m-auto font-bold">
+            APARTMENT25
+            {% if settings.contact_address != blank %}
+              <div>
+                {{ settings.contact_address }}
+              </div>
+            {% endif %}
           </div>
-        {% endif %}
-        </div>
 
-        <div class="grid-item item-s-12 item-m-4">
-        {% if settings.contact_email != blank %}
-          <div>
-            <a href="mailto:{{ settings.contact_email }}">{{ settings.contact_email }}</a>
-          </div>
-        {% endif %}
-        {% if settings.mailchimp_action != blank and settings.mailchimp_hidden != blank %}
-          <form action="{{ settings.mailchimp_action }}" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" target="_blank">
-            <input placeholder="your email address" type="email" name="EMAIL" id="mce-EMAIL">
+          <nav class="grid-item item-s-12 item-m-auto no-gutter grid-row">
+            {% if collections.'men' != blank %}
+            {% assign collection = collections.'men' %}
+            {% include 'nav-collection' %}
```

Comment:
is it weird that this triggers the submenu in the header?

Reply:
no it shouldnt trigger the submenu in the header

Reply:
fix it if you want